README.md

```diff
@@ -207,6 +209,18 @@ class Welcome_test extends TestCase
 
 See [How to Write Tests](https://github.com/kenjis/ci-phpunit-test/blob/master/docs/HowToWriteTests.md) for details.
 
+## (Optional) How to generate test result report
+
+~~~
+composer require phing/phing --dev
+~~~
+
+Use phpunit-frames.xsl file to replade file vendor/phing/phing/etc/phpunit-frames.xsl _//this change is used to show Assertions number in test report_  
```

Comment:
@biechao Add command instruction like `cp vendor/kenjis/ci-phpunit-test/application/tests/phpunit-frames.xsl vendor/phing/phing/etc/`

Reply:
@biechao Add instruction to install `build.xml`.